runner/src/server/index.ts

```diff
@@ -127,6 +130,16 @@ async function createServer(routeConfig: RouteConfig) {
       if ("header" in response && response.header) {
         response.header("X-Robots-Tag", "noindex, nofollow");
 
+        const existingHeaders = response.headers;
+        const existingCsp = existingHeaders["content-security-policy"] || "";
+        if (typeof existingCsp === "string") {
+          const newCsp = existingCsp?.replace(
+            /connect-src[^;]*/,
+            `connect-src 'self' https://${bucketName}.s3.${region}.amazonaws.com/`
+          );
+          response.header("Content-Security-Policy", newCsp);
+        }
```

Comment:
We dynamically append our bucket address to the content security policy, allowing client-side file uploads directly from the users browser, on our domain.

Reply:
does this policy have a lifetime?

Reply:
Not sure I know about this, could you elaborate please?